### ISSUE TITLE
Improve auto config extraction

### ISSUE THREAD

**Issue by jakopako** [jakopako/goskyr#115]
Make it work for http://www.redutajazzclub.cz/program-en

**Comment by jakopako** [comment]
The problem seems to be that each item has a unique set of classes, eg.
- class="progitem col s12 isafter row d01072022 hl2"
- class="progitem col s12 isafter row d02072022 hl2"
- class="progitem col s12 isafter row d03072022 hl2"

**Comment by jakopako** [comment]
For view-source:https://www.brauerei-oerlikon.ch/events/ the auto extraction also doesn't properly work.

**Comment by jakopako** [comment]
Here, https://www.porgy.at/events/upcoming, the item selector is not properly extracted. It works when the path is one node longer, ie 
```
body > div.main > div.main_content > div.container > div.row.events > div.col-md-9.event_list > div.row.event_list
```
instead of
```
body > div.main > div.main_content > div.container > div.row.events > div.col-md-9.event_list
```

**Comment by jakopako** [comment]
last comment will be addressed in new issue #116
